tests/integration/assessment/test_clusters.py

```diff
@@ -22,3 +23,18 @@ def test_cluster_crawler(ws, make_cluster, inventory_schema, sql_backend):
 
     assert len(results) >= 1
     assert results[0].cluster_id == created_cluster.cluster_id
+
+
+def test_cluster_crawler_no_isolation(ws, make_cluster, inventory_schema, sql_backend):
+    created_cluster_2 = make_cluster(data_security_mode=DataSecurityMode.NONE, num_workers=1)
```

Comment:
do we need the suffix `_2`

```suggestion
    created_cluster = make_cluster(data_security_mode=DataSecurityMode.NONE, num_workers=1)
```